web/src/lib/components/settings/Nostr.svelte

```diff
@@ -13,60 +76,70 @@
+    {#if $user.nostrPublicKey && $user.nostrPublicKeyVerified && $user.nostrPublicKey === nostrPublicKey}
+        <div class="w-full flex items-center justify-center mt-8">
             <div>
-                <svg xmlns="http://www.w3.org/2000/svg" class="stroke-current flex-shrink-0 h-6 w-6" fill="none" viewBox="0 0 24 24"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 12l2 2 4-4m6 2a9 9 0 11-18 0 9 9 0 0118 0z" /></svg>
-                <span>We have detected that you have a Nostr extension in your browser, so you'll be able to use your
-                own identity in the entire Plebeian Market.</span>
+                <div class="text-2xl">Your verified Nostr public key is:</div>
+                <div class="flex justify-center items-center gap-4">
+                    <pre class="my-8 text-lg bg-base-300 text-center">{$user.nostrPublicKey}</pre>
+                    <button class="btn btn-secondary" on:click={() => nostrPublicKey = ""}>Change</button>
+                </div>
+                <div>You can use any Nostr client that supports encrypted direct messages (NIP-04) to log in to the Plebeian Market backend.</div>
             </div>
         </div>
-
     {:else}
-        <div class="alert alert-warning shadow-lg">
-            <div>
-                <svg xmlns="http://www.w3.org/2000/svg" class="stroke-current flex-shrink-0 h-6 w-6" fill="none" viewBox="0 0 24 24"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z" /></svg>
-                <span>You don't have a Nostr extension</span>
+        <div class="w-full flex items-center justify-center mt-8">
+            <div class="form-control w-full max-w-lg">
+                <input bind:value={nostrPublicKey} id="nostr-public-key" name="nostr-public-key" type="text" class="bg-transparent z-10 ml-1.5 input input-bordered input-md w-full" />
             </div>
         </div>
-
-        <div class="mt-4 text-justify">
-            <p>You're using your Plebeian Market generated Nostr identity. We created this for you as we detected
-                that you're not using a Nostr browser extension. This is not a problem and you can continue using
-                Plebeian Market this way, but it's highly recommended that you install a Nostr browser extension
-                and create your own Nostr identity so nobody else has access to your private key.</p>
-
-            <p class="mt-3 text-justify">You can use any of this browser extensions:
-                <a class="link" href="https://getalby.com/" target="_blank" rel="noreferrer">Alby</a>,
-                <a class="link" href="https://github.com/fiatjaf/nos2x" target="_blank" rel="noreferrer">nos2x</a> or
-                <a class="link" href="https://www.blockcore.net/wallet" target="_blank" rel="noreferrer">Blockcore</a>.
-            </p>
```

Comment:
I think we must give some information on what Nostr is and how to get an extension. If not here, maybe in the FAQ.

Reply:
Yep, sorry, I missed this. Added to the FAQ.